charts/victoria-metrics-operator/README.md

```diff
@@ -1,6 +1,6 @@
 # Helm Chart For Victoria Metrics Operator.
 
- ![Version: 0.29.5](https://img.shields.io/badge/Version-0.29.5-informational?style=flat-square)
+ ![Version: 0.29.6](https://img.shields.io/badge/Version-0.29.6-informational?style=flat-square)
```

Comment:
Seems like `Chart.yaml` wasn't commited

Reply:
yeah, it was updated in https://github.com/VictoriaMetrics/helm-charts/commit/571d53572c09de33e774afc3761473868bcd0ed3, but doc didn't synced.